From 5a707a4ec3e1fac160d37fde6ac8e0361e8040a9 Mon Sep 17 00:00:00 2001
From: "flinux@vongellhorn.ch" <flinux@vongellhorn.ch>
Date: Thu, 18 Jul 2019 14:17:12 +0200
Subject: [PATCH 1/1] Allow steam to load private libsdl2-2.0.so

This is a quick fix to get steam beta version back running, until sdl2 update gets released

---
 src/intercept/main.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/intercept/main.c b/src/intercept/main.c
index f76c143..e1d0da2 100644
--- a/src/intercept/main.c
+++ b/src/intercept/main.c
@@ -99,6 +99,9 @@ static const char *steam_allowed[] = {
         /* widevine */
         "libwidevinecdmadapter.so",
         "libwidevinecdm.so",
+        
+        /* Temp sdl2 fix */
+        "libSDL2-2.0.so",
 };
 
 static const char *wanted_steam_processes[] = {
-- 
2.22.0

